resources/build/ci/trigger-build-bot.inc.sh

```diff
@@ -187,7 +215,7 @@ function build_bot_update_commands() {
 # Parameters:
 #   1: name of platforms array parameter (byref)
 #
-function build_bot_verify_platforms() {
+function _build_bot_verify_platforms() {
```

Comment:
This was an internal function which should have been underscored from the beginning.